forApple/CaptiveCrypto/CaptiveCrypto/StoredKey/Decipher.swift

```diff
@@ -0,0 +1,62 @@
+// Copyright 2023 VMware, Inc.
+// SPDX-License-Identifier: BSD-2-Clause
+
+import Foundation
+import CryptoKit
+
+extension StoredKey {
+    // Instance methods.
+    func decipher(_ enciphered:Data) throws -> String {
+        switch _storage {
+        case .key:
+            return try decipherWithPrivateKey(enciphered as CFData)
+        case .generic:
+            return try decipherWithSymmetricKey(enciphered)
+        }
+    }
+    func decipher(_ enciphered:Enciphered) throws -> String {
+        return try decipher(enciphered.message)
+    }
+    
+    private func decipherWithSymmetricKey(_ enciphered:Data) throws -> String {
+        let sealed = try AES.GCM.SealedBox(combined: enciphered)
+        let decipheredData = try AES.GCM.open(sealed, using: symmetricKey!)
```

Comment:
Can we avoid force unwrapping and throw error if its nil? Here and other places.

Reply:
Hi, I have removed all the forced unwrapping that could be removed.